### PR TITLE
Daycare operation days

### PR DESCRIPTION
**Pull request by reynders**
#### Summary

Unit operation days data structure and editor: 

Screenshot 2020-11-03 at 10 04 06

#### Checklist for pull request creator

- [ ] The code is consistent with the existing code base
- [ ] Tests have been written for the change (e2e, integration tests)
- [ ] The change has been tested locally
- [ ] The change conforms to the UX specifications
- [ ] The code is self-documenting or has been documented sufficiently, e.g., in the README
- [ ] The branch has been rebased against master before the PR was created

#### Checklist for pull request reviewer (copy to review text box)

```
- [ ] All changes in all changed files have been reviewed
- [ ] The code is consistent with the existing code base
- [ ] Tests have been written for the change (e2e, integration tests)
- [ ] The change has been tested locally
- [ ] The change conforms to the UX specifications
- [ ] The code is self-documenting or has been documented sufficiently, e.g., in the README
- [ ] The PR branch has been rebased against master and force pushed if necessary before merging
```

### CODE REVIEW

File: frontend/packages/employee-frontend/src/components/unit/unit-details/UnitEditor.tsx [hunk 1]
```diff
@@ -703,6 +712,61 @@ export default function UnitEditor(props: Props): JSX.Element {
...
+            onChange={(canApplyPreschool) => updateForm({ canApplyPreschool })}
+          />
+          <Checkbox
+            disabled={!props.editable}
+            label={i18n.unitEditor.field.canApplyClub}
+            checked={form.canApplyClub}
+            onChange={(canApplyClub) => updateForm({ canApplyClub })}
+          />
+        </FixedSpaceColumn>
+      </FormPart>
```
Comment (jaahon) [inline frontend/packages/employee-frontend/src/components/unit/unit-details/UnitEditor.tsx:769]: These probably shouldn't be duplicated here.

**Review by jaahon — COMMENTED** [review]
I think _operational day_ instead of operation day is better English, but it doesn't matter too much.

File: frontend/packages/employee-frontend/src/components/unit/unit-details/UnitEditor.tsx [hunk 2]
```diff
@@ -703,6 +712,61 @@ export default function UnitEditor(props: Props): JSX.Element {
...
+              marginRight={'m'}
+              alignItems={'center'}
+            >
+              <div>{i18n.unitEditor.label.operationDay[day]}</div>
+              <Checkbox
+                disabled={!props.editable}
+                checked={
+                  form.operationDays.some(
+                    (selectedDay) => selectedDay == day
+                  ) ?? false
```
[the remark this answers is not in the document]
Comment (reynders) [reply frontend/packages/employee-frontend/src/components/unit/unit-details/UnitEditor.tsx]: Funny that lint didn't complain about this

**Comment by jaahon** [comment]
We should also take `round_the_clock` units into account in the migration.

**Review by jaahon — APPROVED** [review]

**Pull request merged**